src/components/ad-service-provider.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { DfpAdWrapper } from "./dfp-ad-wrapper";
+
+export function AdServiceProvider(props) {
+  const { adService = "", defaultNetworkID = "" } = props;
```

Comment:
why do you need defaultNetworkId?

Reply:
To have a check for it as in when it's not there then just return null